flask_resources/parsers/url_args.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: This file and ``headers.py`` looks too much alike with only very minor differences, so it must be possible to make them share more code.

For instance, here's the differences between URLArgsParser and HeaderParser:

```python
class URLArgsParser(RequestParser):
     location = 'querystring'
     raw_args_fun = lambda: {**request.headers} 

class HeaderParser(RequestParser):
     location = 'headers'
     raw_args_fun = lambda: request.args.to_dict(flat=False)
```

Reply:
And the decorators can be done similar to this:

```python
def request_parser_decorator(parser_cls, config_attr, context_attr):
    def decorator(f):
        @wraps(f)
        def inner(self, *args, **kwargs):
            parser = select_request_parser(
                self.resource_method,
                getattr(self.resource.config, config_attr),
                parser_cls
            )
            setattr(resource_requestctx, context_attr, parser.parse())
            return f(self, *args, **kwargs)
        return inner
    return decorator

url_args_parer = request_parser_decorator(
    URLArgsParser,
    'request_url_args_parser',
    'url_args',
    
)
headers_parser = request_parser_decorator(
    HeaderParser,
    'request_headers_parser',
    'headers'
)
```

Reply:
Nice I will refactor that :)